spec/features/catalog_spec.rb

```diff
@@ -25,6 +25,11 @@
       visit spotlight.search_exhibit_catalog_path(exhibit)
       expect(page).to have_selector '.card-header', text: 'Item visibility'
     end
+
+    it 'has an edit link' do
```

Comment:
Feature tests are relatively slow (especially in the setup).  Can we include the edit link assertion in the test above?  I think the feature is "it draws the page" and there are several assertions that support "the page is drawn correctly".

Reply:
The two admin tests are hitting different routes so I didn't think it made sense to combine them. The test above is for `search_exhibit_catalog_path` and the Edit link test I added is for `exhibit_solr_document_path`

Reply:
Oh, okay!